index.js

```diff
@@ -14,9 +14,9 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell and
-//. PureScript. It depends on and works nicely with [Ramda][]. Sanctuary
-//. makes it possible to write safe code without null checks.
+//. Sanctuary is a functional programming library inspired by Haskell
+//. and PureScript. It's stricter and more opinionated than [Ramda][].
+//. Sanctuary makes it possible to write safe code without null checks.
```

Comment:
@davidchambers looks good, but maybe we should mention that Sanctuary can/should be used as an alternative to Ramda. On the top of my head: "It includes much the same functionality as Ramda, but is stricter and more opinionated."

I think this would make it more clear what functionality people can expect from Sanctuary ("write safe code without null checks" can be a bit vague). Most likely developers diving into functional programming in javascript will take a look at both Ramda and Sanctuary and I think a wording along these lines would make the relationship between them clear.

Just my 2 cents.